apps/prairielearn/elements/pl-external-grader-variables/pl-external-grader-variables.py

```diff
@@ -28,7 +28,7 @@ def prepare(element_html: str, data: pl.QuestionData) -> None:
 
             html_variables.append(var_dict)
 
-        elif child.tag is lxml.etree.Comment:
+        elif isinstance(child.tag, lxml.etree._Comment):
```

Comment:
This isn't correct, is it? Shouldn't it be this?

```suggestion
        elif isinstance(child, lxml.etree._Comment):
```

Reply:
Oops! [783a1eb](https://github.com/PrairieLearn/PrairieLearn/pull/11243/commits/783a1eb2ecaea8a06baca2ae4d52db3d979180be)